cmd/podman/images/scp.go

```diff
@@ -374,8 +336,100 @@ func parseArgs(args []string, cfg *config.Config) (map[string]config.Destination
 				return nil, err
 			}
 		}
-		scpOpts.URI = append(scpOpts.URI, urlT)
-		scpOpts.Iden = append(scpOpts.Iden, iden)
+		sshInfo.URI = append(sshInfo.URI, urlT)
+		sshInfo.Identities = append(sshInfo.Identities, iden)
 	}
 	return serv, nil
 }
+
+// execLoad executes the podman load command given the podman binary
+func execLoad(podman string) error {
+	var loadCommand []string
+	quiet := ""
+	if source.Quiet {
+		quiet = "-q "
+	}
+	loadCommand = strings.Split(fmt.Sprintf("load %s--input %s", quiet, dest.File), " ")
+	if rootless.IsRootless() {
+		cmd := exec.Command(podman)
+		cmd.Args = append(cmd.Args, parentFlags...)
```

Comment:
There should be a shared function for this, I see you cut and pasted the same code at least 4 times.